handler/src/main/java/io/netty/handler/ssl/Java8SslUtils.java

```diff
@@ -69,7 +70,7 @@ static List getSniHostNames(List<String> names) {
         }
         List<SNIServerName> sniServerNames = new ArrayList<SNIServerName>(names.size());
         for (String name: names) {
-            sniServerNames.add(new SNIHostName(name));
+            sniServerNames.add(new SNIHostName(name.getBytes(StandardCharsets.UTF_8)));
```

Comment:
StandardCharsets are not available in Java 1.6 or 1.7. Use CharsetUtil.

Reply:
@hyperxpro The classname is `Java8SslUtils`, is this properable?

Reply:
Netty 4.1.X compiles at Java 1.6